views/blacklist.ejs

```diff
@@ -1,278 +1,79 @@
 <%- include('./partials/header-quiz'); %>
 
-    <main class="container-sm p-3" id="blacklist">
-        <!-- Title and intro -->
-        <section class="mb-5">
-            <h1>Blacklist</h1>
-            <p>Hieronder vind je je blacklist. Deze quotes komen niet meer voor in de quiz. Klik op de pen om de
-                opmerkingen te wijzigen, of op het vuilbakje om
-                quotes uit je blacklist te verwijderen.</p>
+<main class="container-sm p-3" id="blacklist">
+    <!-- Title and intro -->
+    <section class="mb-5">
+        <h1>Blacklist</h1>
+        <p>Hieronder vind je je blacklist. Deze quotes komen niet meer voor in de quiz. Klik op de pen om de
+            opmerkingen te wijzigen, of op het vuilbakje om quotes uit je blacklist te verwijderen.</p>
+    </section>
+    <% if (blacklist.length === 0) { %>
+        <!-- No quotes in blacklist -->
+        <section>
+            <p>Nog geen quotes in uw blacklist</p>
         </section>
+    <% } else { %>
```

Comment:
I forgot to say, but I think this is a good idea, to add this.

**Feel free to add this to the favorites page as well!** (Not necessary on character page, because that page is only available if there are favorites.)

Perhaps you could also add extra instruction to make it clear to the user, and make the first part bold for emphasis, but this is just a suggestion:

```suggestion
    <% if (blacklist.length === 0) { %>
        <!-- No quotes in blacklist -->
        <section>
            <p><strong>Nog geen quotes in je blacklist.</strong> Gebruik de blacklist-knop tijdens de quiz om een quote toe te voegen.</p>
        </section>
    <% } else { %>
```